devp2p/src/dummy_protocol.rs

```diff
@@ -0,0 +1,95 @@
+use std::net::SocketAddr;
+use std::sync::Arc;
+use trin_core::portalnet::discovery::{Config as DiscoveryConfig, Discovery};
+use trin_core::portalnet::protocol::PortalnetEvents;
+use trin_core::portalnet::types::{Enr, FindContent, FindNodes, Message, Ping, Request};
+use trin_core::portalnet::U256;
+use trin_core::socket;
+use trin_core::utils::get_data_dir;
+
+use discv5::Discv5ConfigBuilder;
+use rocksdb::{Options, DB};
+
+const LISTEN_PORT: u16 = 9876;
+
+#[derive(Clone)]
+pub struct DummyProtocol {
+    pub discovery: Arc<Discovery>,
+    data_radius: U256,
+}
+
+impl DummyProtocol {
```

Comment:
I wonder if there's some way to refactor the current implementation so that we don't have all this duplicate dummy code. I'm not deep enough in it to see the best solution here, so this doesn't need to be a blocker if you don't come up with something quickly. I'm okay waiting to see how much of a pain maintaining it will be, and dealing with it then.

Reply:
Yeah, this was just experimenting and prototyping some code, when https://github.com/carver/trin/pull/80 is merged, I will update it to use the default PortalProtocol struct